bindings/python/visualize/meshcat_visualizer.py

```diff
@@ -0,0 +1,262 @@
+class MeshcatVisualizer:
+    """
+    A simple wrapper to the meshcat visualizer. The MeshcatVisualizer class is highly inspired by the Pinocchio version
+    of the MeshCat visualizer
+    https://github.com/stack-of-tasks/pinocchio/blob/b134b25f1409f5bf036105b996da2d29c1a66a12/bindings/python/pinocchio/visualize/meshcat_visualizer.py
+    """
+
+    def __init__(self):
+        self.viewer = meshcat.Visualizer()
+        self.traversal = dict()
+        self.model = dict()
+        self.link_pos = dict()
+
+    def __is_mesh(self, geometry_object):
+
+        mesh_path = geometry_object.asExternalMesh().getFileLocationOnLocalFileSystem()
+
+        # Check whether the geometry object contains a Mesh supported by MeshCat
+        if mesh_path == "":
+            return False
+
+        _, file_extension = os.path.splitext(mesh_path)
+        if file_extension.lower() in [".dae", ".obj", ".stl"]:
+            return True
+
+        return False
+
+    def __load_mesh(self, geometry_object):
+
+        mesh_path = geometry_object.asExternalMesh().getFileLocationOnLocalFileSystem()
+
+        # try to import the mesh
+        if mesh_path == "":
+            return None
+
+        _, file_extension = os.path.splitext(mesh_path)
+
+        basename = os.path.basename(mesh_path)
+        file_name = os.path.splitext(basename)[0]
+
+        geometry_object.asExternalMesh().setName(file_name)
+
+        if file_extension.lower() == ".dae":
+            obj = meshcat.geometry.DaeMeshGeometry.from_file(mesh_path)
+        elif file_extension.lower() == ".obj":
+            obj = meshcat.geometry.ObjMeshGeometry.from_file(mesh_path)
+        elif file_extension.lower() == ".stl":
+            obj = meshcat.geometry.StlMeshGeometry.from_file(mesh_path)
+        else:
+            msg = "The following mesh cannot be loaded: {}.".format(mesh_path)
+            warnings.warn(msg, category=UserWarning, stacklevel=2)
+            obj = None
+
+        return obj
+
+    def __apply_transform(self, world_H_frame, solid_shape, viewer_name):
+        world_H_geometry = (world_H_frame * solid_shape.getLink_H_geometry()).asHomogeneousTransform().toNumPy()
+        scale = list(solid_shape.asExternalMesh().getScale().toNumPy().flatten())
+        extended_scale = np.diag(np.concatenate((scale, [1.0])))
+        world_H_geometry_scaled = np.array(world_H_geometry).dot(extended_scale)
+
+        # Update viewer configuration.
+        self.viewer[viewer_name].set_transform(world_H_geometry_scaled)
+
+    def __model_exists(self, model_name):
+
+        if model_name in self.model.keys():
+            return True
+
+        if model_name in self.traversal.keys():
+            return True
+
+        if model_name in self.link_pos.keys():
+            return True
+
+        return False
+
+    def __add_model_geometry_to_viewer(self, model, model_geometry: idyn.ModelSolidShapes,
+                                       model_name, color):
+        import meshcat.geometry
```

Comment:
`meshcat` is already imported in top list, is this necessary? Defining it here is useful for lazy imports, but if you already imported it above maybe also this could be moved there.